hollow/src/main/java/com/netflix/hollow/api/producer/DummyBlobStorageCleaner.java

```diff
@@ -0,0 +1,33 @@
+/*
+ *
+ *  Copyright 2017 Netflix, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ *
+ */
+package com.netflix.hollow.api.producer;
+
+/**
+ * This Dummy blob storage cleaner does nothing
+ */
+public class DummyBlobStorageCleaner extends HollowProducer.BlobStorageCleaner {
```

Comment:
Should this be package private, if its the default implementation that does nothing?

Reply:
Moved this into the `HollowProducer` as a private class. Yes it is the default implementation that does nothing.